test/e2e/Tests/Fixtures/FunctionAppFixture.cs

```diff
@@ -12,166 +9,51 @@ namespace Microsoft.Azure.Durable.Tests.DotnetIsolatedE2E;
-
-            FixtureHelpers.StartProcessWithLogging(this._funcProcess, this._logger);
-
-            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
-            {
-                // ensure child processes are cleaned up
-                _jobObjectRegistry = new JobObjectRegistry();
-                _jobObjectRegistry.Register(this._funcProcess);
-            }
-
-            using var httpClient = new HttpClient();
-            this._logger.LogInformation("Waiting for host to be running...");
-            await TestUtility.RetryAsync(async () =>
-            {
-                try
-                {
-                    var response = await httpClient.GetAsync($"{Constants.FunctionsHostUrl}/admin/host/status");
-                    var content = await response.Content.ReadAsStringAsync();
-                    var doc = JsonDocument.Parse(content);
-                    if (doc.RootElement.TryGetProperty("state", out JsonElement value) &&
-                        value.GetString() == "Running")
-                    {
-                        this._logger.LogInformation($"  Current state: Running");
-                        return true;
-                    }
-
-                    this._logger.LogInformation($"  Current state: {value}");
-                    return false;
-                }
-                catch
-                {
-                    if (_funcProcess.HasExited)
-                    {
-                        // Something went wrong starting the host - check the logs
-                        this._logger.LogInformation($"  Current state: process exited - something may have gone wrong.");
-                        return false;
-                    }
-
-                    // Can get exceptions before host is running.
-                    this._logger.LogInformation($"  Current state: process starting");
-                    return false;
-                }
-            }, userMessageCallback: () => string.Join(System.Environment.NewLine, TestLogs.CoreToolsLogs));
+            case "dotnet-isolated":
+                this.functionAppProcess = new IsolatedFunctionAppProcess(this.logger, this.TestLogs);
+                this.functionLanguageLocalizer = new IsolatedTestLanguageLocalizer();
+                break;
+            case "powershell":
+                this.functionAppProcess = new PowerShellFunctionAppProcess(this.logger, this.TestLogs);
+                this.functionLanguageLocalizer = new PowerShellTestLanguageLocalizer();
+                break;
+            default:
+                this.logger.LogWarning("Environment variable E2E_TEST_FUNCTIONS_LANGUAGE not set, tests configured for dotnet-isolated");
+                this.functionAppProcess = new IsolatedFunctionAppProcess(this.logger, this.TestLogs);
+                this.functionLanguageLocalizer = new IsolatedTestLanguageLocalizer();
+                break;
```

Comment:
Should the default be to not run any tests? I'm wondering if it's misleading to have the default be dotnet-isolated. For example, if we're trying to run the PowerShell tests and E2E_TEST_FUNCTIONS_LANGUAGE isn't set correctly to `powershell` then it seems like it will run .NET Isolated tests? Is that correct? Let me know if the behavior is different.

Reply:
So if "e2e-mssql" runs and passes, then is there a chance that the tests actually ran with the Azure Storage backend? Where would I see the warning that the environment variable is missing in these scenarios?